websites/T/Trakt/dist/metadata.json

```
@@ -0,0 +1,18 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "xKubson_",
+		"id": "404217213873029120"
+	},
+	"service": "Trakt",
+	"description": {
+		"en": "With trakt.tv you can: track what you watch and when and discover what's hot and where you can watch it!"
```

This isn't the official description. Source: metadata 
```suggestion
		"en": "Track what you watch and when. Discover what's hot and where you can watch it. Share comments, ratings and recommendations."
```